.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -40,25 +40,28 @@ body:
   - type: textarea
     attributes:
       label: Versions
-      description: Version of relevant packages.
+      description: Please run the following script and paste the output
       value: |
         ```python
-        import platform; print(platform.platform())
-        import sys; print("Python", sys.version)
-        import astropy; print("astropy", astropy.__version__)
-        import numpy; print("Numpy", numpy.__version__)
-        import erfa; print("pyerfa", erfa.__version__)
         try:
-            import scipy
-            print("Scipy", scipy.__version__)
-        except ImportError:
-            print("Scipy not installed")
-        try:
-            import matplotlib
-            print("Matplotlib", matplotlib.__version__)
-        except ImportError:
-            print("Matplotlib not installed")
+            from astropy.utils.report_info import main; main()
```

Comment:
Still not 100% sold on this. This import won't work until astropy 7.0 is released. Until then, we might just get confused users reporting that our bug report template is broken.

I agree with @astrofrog that https://github.com/astropy/astropy-tools is probably a better place for such scripts, but then the same problem about it not being readily installable.

Reply:
> This import won't work until astropy 7.0 is released.

except for reporters running the main branch, which I suspect wouldn't be a negligible fraction ?

> Until then, we might just get confused users reporting that our bug report template is broken.

Now *I*'m confused: why would anyone report this as broken if there's not visible exception being raised ? (this isn't a rhetorical question, I'm genuinely curious)

Reply:
Actually a lot of users report from stable release. Not everyone installs from `main` even when requested for various reasons (no time, no desire, stuck in old release, etc). Therefore for those users, `from astropy.utils.report_info import main` will not work until we release v7. Hope this clarifies the matter.

Reply:
But this is in a `try-except` block.

Reply:
Oh, oops... Must have missed it in the diff. Let me ponder again then when I get the chance. Thanks for your patience.

Reply:
Would it be more explicit if we now do an `astropy` version check instead of a blind try-except?

Reply:
More explicit yes, but maybe also more fragile, and I'm afraid it would come back biting us with users reporting the script is broken...

Reply:
Well, we would need to test this script in CI anyway. And even then if it is still broken, won't we want to know? Under what condition would the script not break in CI but broken when user runs it? 🤔

Reply:
Since the attribute `astropy.__version__` is generated via `setuptools_scm`, I don't feel confortable taking any bets on exactly what could go wrong, but I don't trust that CI exercises all the ways it can be broken, which should all be weird edge cases that no one ever reported yet.

Reply:
Additionally, I note that
1) the main function in the script has a doctest
2) the exact way it's used in the report template is *not* exercised yet (as far as I know). That's definitely doable if you think it's a good idea.